src/diracx/db/os/job_parameters.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+from diracx.db.os.utils import BaseOSDB
+
+
+class JobParametersDB(BaseOSDB):
+    fields = {
+        "JobID": {"type": "long"},
+        "timestamp": {"type": "date"},
+        "CPUNormalizationFactor": {"type": "long"},
+        "NormCPUTime(s)": {"type": "long"},
+        "Memory(kB)": {"type": "long"},
+        "TotalCPUTime(s)": {"type": "long"},
+        "MemoryUsed(kb)": {"type": "long"},
+        "HostName": {"type": "keyword"},
+        "GridCE": {"type": "keyword"},
+        "ModelName": {"type": "keyword"},
+        "Status": {"type": "keyword"},
```

Comment:
I think `Status` should be removed: https://github.com/DIRACGrid/DIRAC/issues/7191
And a few of these should be moved to ElasticPilotParameters: https://github.com/DIRACGrid/DIRAC/pull/7164

Do you want to create this implementation anyway now for testing? In this case put some ToDO notes (the links above would suffice).